.ci/docker-compose.test.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+
+services:
+  test:
+    image: ${TEST_IMAGE_TAG:-test}
+    depends_on: 
+      - database
+    environment:
+      PARALLEL_WORKERS: 1 # makes tests run sequentially instead of simultaneously
+    command: >

```

Comment:
The approach we took in po8klasie/po8klasie and po8klasie/po8klasie-api repo is that we had two entrypoints files (one prod-ready, one for linters and testing) and we would overwrite command here to choose entrypoint for testing.

Why this approach might be better?
IMO it would better if linting work within the docker, cause 1. it'll have the exact version as files in repo, 2. starting another action took pricey minutes, 3. it would cancel this job (so cancel tests) if linting fails.

Reply:
I mean expressing commands directly in yaml here is fine. It just mostly about linting within container